### ISSUE TITLE
Check if event is part of a synced project before sending it to the processor

### ISSUE THREAD

**Issue by ymoriaud** [ynput/ayon-aquarium#7]
The leecher is receiving all events that happen on Aquarium. Some of them might not be inside a synced project.

To avoid unnecessary event to process, we can check if the event is part of a synced project by using the `event.get_context()` un the callback function:

https://github.com/ynput/ayon-aquarium/blob/ea0e891187e601675408219b7ae54157701666b3/services/leecher/leecher/leecher.py#L31-L50

Here is a not tested example:

```python
context = event.get_context()
project = context["project"]
if project is not None and project.data.ayonProjectName is not None:
    # Process the event
```

**Comment by tweak-wtf** [comment]
this is implemented as part of https://github.com/ynput/ayon-aquarium/pull/8

**Comment by ymoriaud** [comment]
Closed by #8
